src/Symfony/Component/Notifier/Bridge/Twitter/TwitterOptions.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Twitter;
+
+use Symfony\Component\Mime\Part\File;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+final class TwitterOptions implements MessageOptionsInterface
+{
+    public const REPLY_MENTIONED_USERS = 'mentionedUsers';
+    public const REPLY_FOLLOWING = 'following';
+
+    private array $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function toArray(): array
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return null;
+    }
+
+    /**
+     * @param string[] $choices
+     *
+     * @return $this
```

Comment:
Just a question, is this `@return` annotation really needed anymore?

Reply:
Yes it is: an API that returns `$this` means that using the return value is *not* mandatory, whereas an API that only returns `static` means one *must* store the return value.